templates/index/login.html.twig

```diff
@@ -1,3 +1,52 @@
+<!--
+* This file is part of the ESO system.
+*
+* (c) SBSystem 2018
+*
+* For the full license information please see LICENSE.md at https://github.com/SBSystem/ESO/LICENSE.md
+-->
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="keywords" content="Elektroniczny dziennik, e-dziennik, dziennik, eso, szkola">
+        <meta name="description" content="Elektroniczny dziennik ESO pozwoli Ci sprawdzać postępy twojego dziecka w nauce, obecności na lekcjach i wiele innnych!">
+        <title>ESO</title>
```

Comment:
Tytuł ESO :: Logowanie, a najlepiej podawane do skryptu przez PHP. Zamień to na zmienną.